src/pages/projects/common/components/CreateProjectModal.tsx

```diff
@@ -142,7 +142,6 @@ export function CreateProjectModal(props: Props) {
               clearButton
               onClearButtonClick={() => handleChange('client_id', '')}
               errorMessage={errors?.errors.client_id}
-              staleTime={Infinity}
```

Comment:
@beganovich @turbo124 Setting staleTime to Infinity manually for the Create Project page is no longer necessary because we have already changed it to be set by default.